Protein/examples/0.9-DRAFT/DisProt_jsonld.json

```diff
@@ -1 +1 @@
-../../../DataRecord/examples/0.2-DRAFT/DisProt_jsonld.json
\ No newline at end of file
+../../../DataRecord/examples/0.2-DRAFT_examples/DisProt_jsonld.json
```

Comment:
Same suggestion here (see previous comment) and for the rest of the links encoded as a json file rather than exposed as a link